packages/client/src/links/wsLink.ts

```diff
@@ -120,6 +120,14 @@ export function createWSClient(opts: WebSocketClientOptions) {
     }
   }
 
+  function closeActiveSubscriptions() {
+    Object.values(pendingRequests).forEach((req) => {
+      if (req.type === 'subscription') {
+        req.callbacks.complete();
+      }
+    });
+  }
```

Comment:
Could you add a test for this?

Reply:
I.e. making sure that if you have an active sub & call `.close()` it should just close

Reply:
Will give it a shot! What's the purpose/differentiation between the interop tests and the usual ones? At first glance I got the impression they are just a duplication ?

Reply:
Is there a reason for not merging? I have added/edited tests to account for the new functionality. I think it should be correct😊👍